pkg/nmi/server/redirector_windows.go

```diff
@@ -0,0 +1,141 @@
+// +build windows
+
+package server
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	v1 "k8s.io/api/core/v1"
+	types "k8s.io/apimachinery/pkg/types"
+	"k8s.io/klog"
+)
+
+type WindowsRedirector struct {
+	Server *Server
+}
+
+func makeRedirectorInt(server *Server) RedirectorInt {
+	return &WindowsRedirector{Server: server}
+}
+
+var podMap = make(map[types.UID]string)
```

Comment:
Suggest moving this map to Sync() so that access to this map is localized to Sync() method to avoid possible concurrency issues